cmd/soroban-rpc/internal/methods/get_events.go

```diff
@@ -42,19 +37,18 @@ type EventInfoValue struct {
 
 type GetEventsRequest struct {
 	StartLedger int32              `json:"startLedger,string"`
-	EndLedger   int32              `json:"endLedger,string"`
 	Filters     []EventFilter      `json:"filters"`
 	Pagination  *PaginationOptions `json:"pagination,omitempty"`
 }
 
 func (g *GetEventsRequest) Valid() error {
-	// Validate start & end ledger
-	// Validate the ledger range min/max
-	if g.EndLedger < g.StartLedger {
-		return errors.New("endLedger must be after or the same as startLedger")
+	// Validate start
+	// Validate the paging limit (if it exists)
+	if g.StartLedger <= 0 {
+		return errors.New("startLedger must be positive")
 	}
-	if g.EndLedger-g.StartLedger > maxLedgerRange {
-		return fmt.Errorf("endLedger must be less than %d ledgers after startLedger", maxLedgerRange)
+	if g.Pagination != nil && g.Pagination.Limit > maxLimit {
+		return fmt.Errorf("limit must not exceed %d", maxLimit)
```

Comment:
Should make sure we update the docs for this as well, as currently the `limit` part is 200 (iirc)